src/Microsoft.ML.Transforms/Text/StopWordsRemoverTransform.cs

```diff
@@ -722,30 +745,10 @@ private void LoadStopWords(IHostEnvironment env, IChannel ch, ArgumentsBase load
                 }
                 ch.CheckUserArg(stopWordsMap.Count > 0, nameof(Arguments.Stopwords), "stopwords is empty");
             }
-            else if (Utils.Size(loaderArgs.Stopword) > 0)
-            {
-                bool warnEmpty = true;
-                foreach (string word in loaderArgs.Stopword)
-                {
-                    var stopword = word.AsSpan();
-                    stopword = stopword.Trim(' ');
-                    if (!stopword.IsEmpty)
-                    {
-                        buffer.Clear();
-                        ReadOnlyMemoryUtils.AddLowerCaseToStringBuilder(stopword, buffer);
-                        stopWordsMap.Add(buffer);
-                    }
-                    else if (warnEmpty)
-                    {
-                        ch.Warning("Empty strings ignored in 'stopword' specification");
-                        warnEmpty = false;
-                    }
-                }
-            }
             else
             {
-                string srcCol = loaderArgs.StopwordsColumn;
-                var loader = LoadStopwords(env, ch, loaderArgs.DataFile, loaderArgs.Loader, ref srcCol);
+                string srcCol = stopwordsColumn;
+                var loader = GetLoaderForStopwords(ch, dataFile, loaderFactory, ref srcCol);
```

Comment:
>GetLoaderForStopwords [](start = 29, length = 21)

can we factor it differently? have a `LoadStopWords(dataFile, loaderFactory, stopwordsColumn, NormStr.Pool target)` and put the cursoring loop there? Or even not the cursoring loop, but a `GetColumn<ROM<char>>`?